### PR TITLE
linkerd: 2020-05-01 -> edge-21.6.2

### LINKED ISSUE #113218: linkerd-unstable contains no binary
**Describe the bug**
The output of linkerd-unstable contains no binaries

**Notify maintainers**

@Gonzih

### PR DESCRIPTION
**Pull request by superherointj**
linkerd.stable: init stable-2.9.5
linkerd.edge: init edge-21.6.2

Fixes #113218

This PR was a huge colaboration between many people: @NobbZ, @kalbasit, @Tomberek, @ldesgoui, @Gonzih, @jnetod, @bryanasdev000, @AndersonTorres and more...

And @jnetod won the crown by being the last one to put the hardest nail in the coffin!

### CODE REVIEW

**Comment by AndersonTorres** [comment]
__Thou shall never start version attributes with non-numerical characters__

**Comment by superherointj** [comment]
> _Thou shall never start version attributes with non-numerical characters_

Fixed.

**Comment by superherointj** [comment]
This PR likely should be backported to stable at least. Because Linkerd is broken there. And shouldn't be.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127130` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>linkerd (linkerd_edge)</li>
<li>linkerd_stable</li>
</ul>
</details>

**Review by bryanasdev000 — APPROVED** [review]
Freaking awesome!

Tested it on a local cluster with some focus on template things, all working as expected.

+1 from me for backporting it, 21.05 do not need to keep a broken linkerd.

Thanks guys!

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by superherointj** [comment]
@SuperSandro2000 Is it possible to automatically backport this PR to stable? Should I add some label and the bot will do it? Is it okay?

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127130` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>linkerd</li>
<li>linkerd_stable</li>
</ul>
</details>

**Review by bryanasdev000 — APPROVED** [review]
LGTM.

**Review by bryanasdev000 — APPROVED** [review]
